client/src/components/ContestList/ContestList.tsx

```diff
@@ -8,46 +9,51 @@ import Container from '@material-ui/core/Container';
 import Button from '@material-ui/core/Button';
 import useStyles from './useStyles';
 import { Contest } from '../../interface/User';
-import LionTatoo from "../../Images/lionTatoo.png";
+import LionTatoo from '../../Images/lionTatoo.png';
 
 interface Props {
-    userContests: Contest[];
+  userContests: Contest[];
 }
 
 export default function ContestList({ userContests }: Props): JSX.Element {
-    const classes = useStyles();
+  const classes = useStyles();
 
-    return (
-        <>
-            {/* Write ternary operator checking to see if any data has been recieved, otherwise return default message */}
-            {userContests.length > 0 ? (
-
-                <List className={classes.list}>
-                    {/* map through given data and alter the return inside of the map */}
-                    {userContests.map(contest => (
-                        <>
-                            {console.log(contest)}
-                            <ListItem key={contest.title}>
-                                <Avatar className={classes.avatar} alt="Test Contest Image" variant="square" src={LionTatoo} />
-                                <Grid className={classes.grid} container>
-                                    <ListItemText
-                                        primary={<>
-                                            <Typography className={classes.typography}>{contest.title}</Typography>
-                                        </>}
-                                        secondary={<>
-                                            <Typography>{contest.description}</Typography>
-                                            <Container className={classes.container}>
-                                                <Button className={classes.button}>{contest.prizeAmount}</Button>
-                                            </Container>
-                                        </>} />
-                                </Grid>
-                            </ListItem>
-                        </>
-                    ))}
-                </List>
-
-            ) : (<Typography> None Available </Typography>)}
-
-        </>
-    )
-}
\ No newline at end of file
+  return (
+    <>
+      {/* Write ternary operator checking to see if any data has been recieved, otherwise return default message */}
+      {userContests.length > 0 ? (
+        <List className={classes.list}>
+          {/* map through given data and alter the return inside of the map */}
+          {userContests.map((contest) => (
+            <>
+              <ListItem key={contest.title}>
```

Comment:
Maybe use something more unique as a key here? Because two contests can have the same title

Reply:
This is fixed.

Reply:
This is fixed.